app/routers/account.py

```diff
@@ -4,10 +4,17 @@
 from ..core.db import get_db
 from ..schemas.account import AccountCreate
 from ..services import account as account_service
+from app.core.api_key_auth import check_api_token
 
 router = APIRouter()
+secured_router = APIRouter(dependencies=[Depends(check_api_token)])
 
 
 @router.post("/")
 async def create_account(dto: AccountCreate, db: Session = Depends(get_db)):
     return await account_service.create_account(dto, db)
+
+
+@router.post("/signin")
+async def account_signin(dto: AccountCreate, db: Session = Depends(get_db)):
```

Comment:
dto should be different. Create a dedicated dto for signin. It should have email and password.

Reply:
This is still an issue